udata/core/organization/views.py

```diff
@@ -76,8 +76,8 @@ def get_context(self):
         if self.organization.deleted and not can_view.can():
             abort(410)
 
-        datasets = Dataset.objects(organization=self.organization).visible()
-        reuses = Reuse.objects(organization=self.organization).visible()
+        datasets = Dataset.objects(organization=self.organization).order_by('-temporal_coverage.end', '-metrics.reuses', '-metrics.followers').visible()
+        reuses = Reuse.objects(organization=self.organization).order_by('-metrics.reuses', '-metrics.followers').visible()
```

Comment:
There is a `metrics.reuses` on a `Reuse` object?

Reply:
Yes.

Not sure `metrics.reuses` exists but these elements are available for display.

Reply:
I agree with @abulte : there is no `metrics.reuses` on the `Reuse` objects (only on datasets et organizations), but there is instead a `metrics.datasets`